### PR TITLE
Updates example section in the Readme

### PR DESCRIPTION
**Pull request by Trybnetic**
As outlined in #47, this PR updates the documentation in the readme. Specifically, I made the examples consistent in the sense that they are both using the `fruits.txt` dataset now and extended the descriptions of the examples a bit. 

Changes introduced in this PR:
- Consistent use of  `fruits.txt` in the examples
- Extended descriptions of the examples
- Overview of the function and its parameters in the readme
- Updated the CI status badge

I see some further changes that could be included in this PR:
- [ ] Clean up (e.g. the no longer used PNGs, the deprecated arguments mentioned in the readme, etc.)
- [ ] Longer explanation on the use of `leftWeight` and `rightWeight`

### CODE REVIEW

**Comment by Pierre-Sassoulas** [comment]
Hey @Trybnetic, is this still a work in progress ?

**Comment by Trybnetic** [comment]
I actually struggled with explaining on the use of `leftWeight` and `rightWeight`, but I will try the next days to do the mentioned clean up and then this PR should be ready to be merged.

File: README.md [hunk 1]
```diff
@@ -92,83 +99,77 @@ colorDict = {
...
-6,John,drinks,8.0
-7,Ben,bread,2.0
-8,Mike,bread,3.5
-9,John,meat,13.0
-```
+However, not always you have or can have the data available in the format mentioned in
+the previous example (e.g. if the dataset is too large). In this case, the weights
+between the true and predicted labels can also be calculated beforehand and used to
+create the sankey diagram. In this example, we continue to work with the data loaded
+already in the previous example:
```
Comment (Pierre-Sassoulas) [inline README.md:128]:
```suggestion
However, the data may not always be available in the format mentioned in
the previous example (for instance, if the dataset is too large). In such cases, the weights
between the true and predicted labels can be calculated in advance and used to
create the Sankey diagram. In this example, we will continue working with the data that was loaded
in the previous example:
```

File: README.md [hunk 2]
```diff
@@ -4,14 +4,17 @@ Uses matplotlib to create simple <a href="https://en.wikipedia.org/wiki/Sankey_d
...
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 
-## Example
+## Examples
 
-With fruits.txt :
+### Simple expected/predicted example with fruits.txt:
+
+`pysankey` contains a simple expected/predicted dataset called `fruits.txt` which looks
+the following:
```
Comment (Pierre-Sassoulas) [inline README.md:17]:
```suggestion
`pysankey` contains a simple expected/predicted dataset called `fruits.txt` which looks
like the following:
```

**Review by Pierre-Sassoulas — COMMENTED** [review]
Thank you, made some suggestions, let me know what you think and let's merge this.

**Comment by Trybnetic** [comment]
Both suggestions look very good to me! Thanks for the proof-read!

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**